docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,649 @@
+for setting it up, and any other information the developer(s) would like to provide to the
+end user.
+
+#### Frontend
+
+node_modules/ is a directory which houses all of the JavaScript packages you'll need. You should
+never have do anything with that directory (other than deleting it if you're getting weird issues,
+but that should be __*RARE*__). This will appear after you've run `npm install` later on. This
+directory is also usually not included in a Git repository.
+
+public/ is a directory to store unique files and images like manifest files, HTML files, or
+special image files. They're put here for easy reference and access.
+
+src/ is the directory with your actual project files. All of your React components,
+views, tests, and code will be inside here.
+
+package.json is a file containing dependency/package information, scripts to be run using npm,
+and other data about your project.
+
+package-lock.json holds information on dependencies which dependencies require and special 
+information that should rarely change. You should never have to do anything with this file, and
+it should **_definitely_** be included in a Git repository.
+
+tsconfig.json holds information about your Typescript configuration.
+
+#### Backend
+
+pom.xml is a file with project build instructions, dependencies, and other project information.
+
+api/ is a directory handling routing information and data transfer objects (DTOs).
+
+persist/ is a directory for handling and saving data.
+
+service/ is the entry point for your project, so you'll start it up from there. It handles
+connecting everything together and basically 'running' your project.
+
+### What will each week look like?
+
+Every week, we'll have one of these meetings where we'll go over new concepts that
+you'll need to build your project. After the meetings, we may give out a description, starter
+code, and other resources for what we think you should work on during the upcoming week. 
+You're always free to work on other parts of your project or not do it at all if that's
+what you want to do. The following weekend before the next meeting, we'll send out
+what we're calling 'catch-up code', which is our implementation of the concepts and
+descriptions we told you to work on during the week. This catch-up code will show
+the current state of the project and the progression of our app over time. The
+reason we're doing this is because we understand that there are weeks that you may 
+be sick, drowning in work, just not in the mood to work on your project, or stuck on some 
+issues, but you'll probably want to stay caught up with the material. This is our way of 
+ensuring everyone has the opportunity to be on the same page.
+
+### Important Links
+
+Just in case you don't have the links or dont't know where they are, here are some 
+important course resources.
+
+>Note: there is also a dictionary of common terms we've compiled for you which should be
```

Comment:
ohhh I see this is here - maybe move it up a little

Reply:
I added a quick thing up above.